asciidoc-release-notes/README.adoc

```diff
@@ -0,0 +1,10 @@
+== SUSE Product 1.0 (SP 1.0) Release Notes
+
+== How to contribute
+
+For a comprehensive overview, see https://confluence.suse.com/display/documentation/Contributing+to+release+notes.
+
+1. Create a new branch with name that includes: your username, the original branch name, and descriptive suffix (e.g. `lkucharczyk-15_SP3-newfeature`).
```

Comment:
This looks like a Markdown list in an AsciiDoc file ... AsciiDoc uses just `. ` instead of `n. `

Reply:
Fixed.